test/platform/azure/__snapshots__/index.spec.ts.snap

```diff
@@ -21,7 +21,6 @@ Object {
   "autoCompleteSetBy": Object {
     "id": 123,
   },
-  "branchName": "some-branch",
```

Comment:
NOTE: This is handled in `getRenovatePRFormat` in azure-helpers, but not in the azure file directly.